packages/playground/src/components/pull-request/sections/pull-request-check-section.tsx

```diff
@@ -64,39 +75,71 @@ const PullRequestCheckSection = ({ checkData, checksInfo }: PullRequestMergeSect
                           : check.check.status === ExecutionState.PENDING
                             ? 'Pending...'
                             : `Errored in ${time}`}
-                  </div>
+                  </Text>
                 </div>
-                <div className="grid grid-cols-[84px_auto] items-center">
-                  <div className="col-span-1">
-                    {/* TODO: figure out how to do link in this? */}
-                    {/* <Link
-        className="text-blue-500 mx-2"
-        to={
-          routes.toCODEPullRequest({
-            repoPath: repoMetadata.path as string,
-            pullRequestId: String(pullReqMetadata.number),
-            pullRequestSection: PullRequestSection.CHECKS
-          }) + `?uid=${check.check.identifier}`
-        }
-      ></Link> */}
-                    {check.check.status !== ExecutionState.PENDING && (
-                      <Text weight="medium" size={1}>
-                        Details
+                <div className="flex items-center gap-4">
+                  <Text size={1} color="tertiaryBackground">
```

Comment:
so i made this into a grid because in gitness they wanted to keep the elements in the position, cause now it'll jump position if one item is not there technically @joetaylor-harness